spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -138,4 +138,14 @@ def submit_current_password_and_totp
     fill_in 'code', with: generate_totp_code(@secret)
     click_submit_default
   end
+
+  def sign_up_and_2fa_user
```

Comment:
we were only using it in two places, and having the steps in a different file was proving to be confusing / obscured what was happening